### PR TITLE
add test and lint

### PR DESCRIPTION
**Pull request by Inveracity**

### CODE REVIEW

**Comment by Inveracity** [comment]
Tests pass but lint fails
let me know if that needs to be split up in two separate actions

File: blackbox/mixins.py [hunk 1]
```diff
@@ -26,7 +26,6 @@ def _get_all_connstrings() -> list:
         """Get all connstrings in the config."""
         return list(chain(
             Blackbox.databases,
-            Blackbox.loggers,
```
Comment (lemonsaurus) [inline blackbox/mixins.py]: I think this change is already in master, was added by #18

File: tests/test_databases.py [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
+        ]
+        Postgres()
+
+# Not yet implemented
+# def test_redis_handler_can_be_instantiated(config_file):
+#     """Test if the Redis database handler can be instantiated."""
+
+#     from blackbox.handlers import Redis
+
+#     Redis()
```
Comment (lemonsaurus) [inline tests/test_databases.py]: This is actually implemented now, so we can probably uncomment it.

It was implemented in #18

**Review by lemonsaurus — CHANGES_REQUESTED** [review]
Looks great! A few comments.

Are you sure `tests/__init__.py` is necessary?

**Comment by Inveracity** [comment]
> Looks great! A few comments.
> 
> Are you sure `tests/__init__.py` is necessary?

yeah pytest requires the init

File: tests/test_databases.py (hunk 2 above)
Comment (lemonsaurus) [reply tests/test_databases.py]: Looks like you fixed this in the latest merge.

File: blackbox/mixins.py (hunk 1 above)
Comment (lemonsaurus) [reply blackbox/mixins.py]: Looks like this was also solved in the latest merge.

**Review by lemonsaurus — APPROVED** [review]
Excellent! This looks great now.

**Pull request merged**